backend/windmill-common/src/utils.rs

```diff
@@ -120,6 +132,33 @@ lazy_static::lazy_static! {
     };
 }
 
+lazy_static::lazy_static! {
+    pub static ref AGENT_TOKEN: String = std::env::var("AGENT_TOKEN").unwrap_or_default();
+}
+
+#[derive(Clone, Debug, PartialEq)]
+#[non_exhaustive]
+pub enum AgentWorkerSuffix {
+    EnableLiveShell,
+}
+
+impl TryFrom<&str> for AgentWorkerSuffix {
+    type Error = Error;
+    fn try_from(value: &str) -> std::result::Result<Self, Self::Error> {
+        let suffix = match value {
+            "enable.live.shell" => AgentWorkerSuffix::EnableLiveShell,
+            suffix => {
+                return Err(Error::Anyhow {
+                    error: anyhow::anyhow!("Unknown suffix: {}", suffix),
+                    location: "utils.rs@149".to_string(),
+                })
+            }
+        };
+
+        Ok(suffix)
+    }
+}
```

Comment:
what is the point of all this, why do we need an enable.live.shell and why would it be in the suffix